corporate/views/upgrade.py

```diff
@@ -146,6 +147,15 @@ def initial_upgrade(
         "manual_license_management": manual_license_management,
     }
 
+    # Likely, user was successful in adding a card and we are rendering the page again.
+    if customer is not None:
+        assert customer.stripe_customer_id is not None
+        stripe_customer = stripe_get_customer(customer.stripe_customer_id)
+        payment_method = payment_method_string(stripe_customer)
+        # For other cases, we show add card button.
+        if "ending in" in payment_method:
+            context["payment_method"] = payment_method_string(stripe_customer)
```

Comment:
`context["payment_method"] = payment_method`. I think we can avoid another call to `payment_method_string()` ?

Reply:
oh yeah! Thanks for noticing.